examples/niswitch_control_relays/measurement.py

```diff
@@ -101,12 +111,13 @@ def _create_niswitch_session(
         )
 
     # This uses the topology configured in MAX.
-    return niswitch.Session(session_info.resource_name, **session_kwargs)
+    return niswitch.Session(session_info.resource_name, options=options, **session_kwargs)
```

Comment:
`niswitch.Session` is different from the others. Instead of `options`, I think we need to pass `simulate=True` and `topology="2567/Independent"`.

Reply:
https://github.com/ni/nimi-python/issues/1959